apps/server/src/controllers/integrationController.ts

```diff
@@ -222,6 +237,74 @@ export function dispatchFromAdapter(type: string, payload: unknown, source?: 'os
       return { topic: 'timer', payload: timer };
     }
 
+    // ontime/change/{eventID}/{propertyName}
+    case 'change': {
+      try {
+        if (params.length < 2) {
+          throw new Error('Invalid parameters');
+        }
+
+        const eventID = params[0];
+        const propertyName = params[1] as keyof OntimeEvent;
+
+        const isKeyOfOntimeEvent = (key: string): key is keyof OntimeEvent => {
+          return key in event;
+        };
```

Comment:
it would be nice to abstract this utility to the guards file in the types package. or at least, remove it from the closure scope

Reply:
problem about sending it to the types package, is how would I import `event` from there? I'm assuming they're different packages so I wouldnt be able to import like "../../../../..etc" or would I?

Reply:
there is a guards file in the packages, you can create a function there with your guard.
after that you will need to export the function from the index in the types package

From here, you just import the types like any other package, see an example here
https://github.com/cpvalente/ontime/blob/e2aabe36468057b9ee4febc7b389a64a0d1b2f44/apps/server/src/modules/loadDb.ts#L4

Reply:
Yeah that makes sense, the only thing I am confused is 

```
export const isKeyOfOntimeEvent = (key: string): key is keyof OntimeEvent => {
  return key in event;
};
```

needs event which is imported from `/models/eventsDefinition.js`

I could do this, but it just feels very wrong
`import { event } from '../../../../apps/server/src/models/eventsDefinition.js';`

I need the `event` object to be able to do `isKeyOfOntimeEvent`as it is right now